margarethe913/tldraw
# Created tests for zoom lock toggling

Describe what your pull request does. If you can, add GIFs or images showing the before and after of your change.

### Change type


### Test plan

1. Define a default view frame
2. Try to change view frame scale
3. Make sure that changes can't be applied if the lock is toggled


### Release notes

- Fixed a bug with…
